mautic/mautic
# TPROD-291 Remove use of KernelInterface::getName() and the kernel.name parameter

…y name i.e. app



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [n]
| New feature/enhancement? (use the a.x branch)      | [n]
| Deprecations?                          | [y]
| BC breaks? (use the c.x branch)        | [n ]
| Automated tests included?              | [n] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-291



#### Description:
KernelInterface::getName() and the kernel.name parameter is removed from symfony 5 and we are removing use of the same.

#### Steps to test this PR:
1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try executing following command 'bin/console --version’. and check if outputs correctly and not breaking anything.


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11138"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
